### PR TITLE
Port app-scrollpos-control

### PR DESCRIPTION
**Pull request by blasten**

### CODE REVIEW

**Comment by keanulee** [comment]
@blasten Can you rename it to `app-scroll-position` as well?

File: app-scroll-position/demo/index.html [hunk 1]
```diff
@@ -27,56 +27,60 @@
   <link rel="import" href="../../app-header/app-header.html">
   <link rel="import" href="../../app-toolbar/app-toolbar.html">
   <link rel="import" href="../../demo/sample-content.html">
-  <link rel="import" href="../app-scrollpos-control.html">
+  <link rel="import" href="../app-scroll-position.html">
```
Comment (frankiefu) [inline app-scroll-position/demo/index.html]: ```<title>app-scrollpos-control demo</title>```
Can you update the title too?

Comment (blasten) [inline app-scroll-position/demo/index.html]: fixed!

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**